wpeverest/everest-forms
# Enhancement - Export Selected Entries

### All Submissions:


### Changes proposed in this Pull Request:
Previously All the entries are exported. There is no way to export selected entries . This PR enhance this feature.

### How to test the changes in this Pull Request:

1. Go to the entries in admin site of Everest Forms.
2. Select the checkbox of entries table.
3. Click on Export CSV and verify only selected entries is exported or not.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Export Selected Entries.
